lib/handlers/get.js

```diff
@@ -118,23 +118,25 @@ async function handler (req, res, next) {
     let headers = {
       'Content-Type': contentType
     }
+    let statusCode = 200
     if (contentRange) {
       headers = {
         ...headers,
         'Content-Range': contentRange,
         'Accept-Ranges': 'bytes',
         'Content-Length': chunksize
       }
-      res.statusCode = 206
```

Comment:
This needs to be set here on the response because when PREP activates, the status code is still 206 on the response. This code will mean PREP will always use the default `res.statusCode` value of 200 when sending a response.